ci-operator/config/maistra/OWNERS

```diff
@@ -0,0 +1,15 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
+- aslakknutsen
+- bmangoen 
+- jwendell
```

Comment:
Do we need all these people? IMO you could add Daniel (unless he's not on openshift org yet) and remove Aslak and Brian.

Reply:
I had added all members in OWNERS before and the CI presubmit check was asking everyone in the OWNERS should join OpenShift organization. So I updated that. When people come back from PTO, I can add them after joining OpenShift organization. 
I will use the OWNERS_ALIASES file. Thanks.